src/odemis/gui/cont/temperature.py

```diff
@@ -114,13 +119,11 @@ def _updateTemperature(self, temperature):
                 logging.exception("Could not compute the rate of change of temperature.")
                 return
 
-        # logging.debug(u"Temperature: %f °C, speed is %f °C / s", temperature, speed)
-
         # determine setting the warning
         if safe_range is not None and not safe_range[0] <= temperature <= safe_range[1] :
             # temperature is out of range
             # change colour to red
-            wx.CallAfter(self.temperature_label.SetForegroundColour, gui.FG_COLOUR_ERROR)
+            set_label_colour(self.temperature_label, gui.FG_COLOUR_ERROR, True)
```

Comment:
Please, in all these calls write explicitly `bold=True/False`. A lone True/False is really hard to read.